nixos/modules/programs/singularity/suid.patch

```diff
@@ -0,0 +1,55 @@
+diff --git source/src/cmd/singularity/cli/actions.go source/src/cmd/singularity/cli/actions.go
+index d3f46be..381dc6b 100644
+--- source/src/cmd/singularity/cli/actions.go
++++ source/src/cmd/singularity/cli/actions.go
+@@ -261,7 +261,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
+ 	uid := uint32(os.Getuid())
+ 	gid := uint32(os.Getgid())
+ 
+-	starter := buildcfg.LIBEXECDIR + "/singularity/bin/starter-suid"
++	starter := "/run/wrappers/bin/singularity-suid"
```

Comment:
I think we can avoid this patch by moving `singularity-suid` to `singularity-suid.org` in the package and then symlink `$out/libexec/singularity/binstarter-suid` to `/run/wrappers/bin/singularity-suid`. This way we will have an easier time updating singularity.

Reply:
That's a great idea! I didn't realise links from the store to outside the store were allowed.